sdk/src/test/java/com/horizen/account/utils/RlpStreamCodecTest.java

```diff
@@ -0,0 +1,239 @@
+package com.horizen.account.utils;
+
+import com.google.common.primitives.Bytes;
+import com.horizen.utils.BytesUtils;
+import org.junit.Test;
+import org.web3j.rlp.RlpEncoder;
+import org.web3j.rlp.RlpList;
+import org.web3j.rlp.RlpString;
+import org.web3j.utils.Numeric;
+import scorex.util.ByteArrayBuilder;
+import scorex.util.serialization.Reader;
+import scorex.util.serialization.VLQByteBufferReader;
+import scorex.util.serialization.VLQByteBufferWriter;
+import scorex.util.serialization.Writer;
+import java.math.BigInteger;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import static org.junit.Assert.*;
+
+public class RlpStreamCodecTest {
```

Comment:
Pls, take a look at the original implementation tests, maybe we may reuse the, as well: https://github.com/web3j/web3j/blob/master/rlp/src/test/java/org/web3j/rlp/RlpDecoderTest.java
https://github.com/web3j/web3j/blob/master/rlp/src/test/java/org/web3j/rlp/RlpEncoderTest.java

Reply:
Ok, done